frontend/src/pages/LoginRegister/RegisterPage.vue

```diff
@@ -27,7 +27,13 @@
         type="password"
         name="confirmPassword"
         placeholder="Confirm password"
-        :rules="[{ required: true, message: 'Confirm password is required' }]"
+        :rules="[
+          { required: true, message: 'Confirm password is required' },
+          {
+            validator: validatePasswordConfirmation,
```

Comment:
Side note: should we disable the button if any rules are not met?

Reply:
hmm, not sure about this. Seems like most examples still allow you to click the submit button